### PR TITLE
Provider interface displays a list of Inquiries

### PR DESCRIPTION
**Pull request by LucasKnudsen**
[PT Story](https://www.pivotaltracker.com/story/show/178422522)

## User story
```
As a broker
In order to have an overview of my Inquiries
I want to see a list of all Inquiries that's sortable by date
```

## Changes proposed
- Builds simple admin interface with sidebar and header
- Adds table of inquiries
- Adds collapsible functionality to table
- GET request to fetch inquiries
- Can sort by date
- Responsiveness

## Screenshots
https://recordit.co/zp4XqKC6pp NEW

### CODE REVIEW

File: src/components/InquiryRows.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            <TableCell data-cy='start-date'>{item.start_date}</TableCell>
+          </>
+        )}
+      </TableRow>
+      <TableRow>
+        <TableCell
+          colSpan={4}
+          className='collapsible-cell'
+          data-cy='inquiry-collapsible-cell'>
+          <Collapse in={open} timeout='auto' unmountOnExit>
```
Comment (offcast-dev) [inline src/components/InquiryRows.jsx:29]: what is timeout='auto'? shouldn't it have no timeout?

File: src/styles/styles.scss [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+.table {
+  margin-top: 50px;
+  height: auto;
+  @media only screen and (max-width: $smallScreen) {
+    max-width: 600px;
+  }
+
+  .table-header th {
```
Comment (offcast-dev) [inline src/styles/styles.scss:53]: what is th?

File: src/styles/styles.scss [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  position: fixed;
+  top: 0;
+  z-index: 1500;
+  height: 75px;
+  width: 100%;
+  background-color: $secondaryColor;
+
+  h4 {
+    color: white;
+    font-size: 2rem;
```
Comment (offcast-dev) [inline src/styles/styles.scss:16]: I understand 1rem = 16px. Maybe we should stick to one

**Review by offcast-dev — COMMENTED** [review]
Looks good, but I think it is better to change sorting switch to other element. Now you cannot really understand what it is doing and in which position.
Check this one out
https://material-ui.com/components/tables/#sorting-amp-selecting

File: src/components/InquiryRows.jsx (hunk 1 above)
Comment (LucasKnudsen) [reply src/components/InquiryRows.jsx:29]: Duration of the transition

File: src/styles/styles.scss (hunk 2 above)
Comment (LucasKnudsen) [reply src/styles/styles.scss:53]: HTML element for table headers

File: src/styles/styles.scss (hunk 3 above)
Comment (LucasKnudsen) [reply src/styles/styles.scss:16]: Like 1rem for the header

Comment (offcast-dev) [reply src/styles/styles.scss:16]: Sorry, didn't convey myself properly. I meant to use 32px instead of 2rem. Or define rules where we use rems where px

**Review by offcast-dev — APPROVED** [review]

**Review by arunbhalli — APPROVED** [review]

**Pull request merged**